### PR TITLE
React stopwatch component

### PR DESCRIPTION
**Pull request by easton-morris**
reactStopwatch

I guess we can't use increment notation in JS? has to be a pure expression?

### CODE REVIEW

File: react-stopwatch-component/src/stopwatch.jsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      console.log('clear ran');
+    }
+  }
+
+  tick() {
+    let newTime = this.state.time + 1;
+    console.log(newTime);
+    this.setState(prevState => (
+      { time: newTime }
+    ));
```
Comment (scbowler) [inline react-stopwatch-component/src/stopwatch.jsx:31]: Using the callback function for `setState` here is overkill, it is better to just pass `setState` an object:

```JSX
this.setState({ time: newTime });
```

File: react-stopwatch-component/src/stopwatch.jsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    this.setState(prevState => ({
+      paused: !prevState.paused
+    }));
+  }
+
+  faceClick() {
+    if (this.state.paused === true) {
+      this.setState(prevState => ({
+        time: 0
+      }))
```
Comment (scbowler) [inline react-stopwatch-component/src/stopwatch.jsx:46]: Here just pass `setState` an object:

```JSX
this.setState({ time: 0 });
```

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good but a few updates needed. Also remove all `console.log`s before submitting pull requests. console logs are for testing during development only and should not be left in your final solution. Let me know if you have any questions.

File: react-stopwatch-component/src/stopwatch.jsx (hunk 2 above)
Comment (easton-morris) [reply react-stopwatch-component/src/stopwatch.jsx]: fixed!

File: react-stopwatch-component/src/stopwatch.jsx (hunk 1 above)
Comment (easton-morris) [reply react-stopwatch-component/src/stopwatch.jsx]: fixed!

**Comment by easton-morris** [comment]
console.logs accidentally got left in because i was so used to the lintr yelling at me for them. i was working on base IDE haha

**Pull request merged**